tests/test_cnn.py

```diff
@@ -7,18 +7,18 @@
 from stable_baselines3.common.identity_env import FakeImageEnv
 from stable_baselines3.common.utils import zip_strict
 
-from sb3_contrib import TQC
+from sb3_contrib import QRDQN, TQC
 
 
-@pytest.mark.parametrize("model_class", [TQC])
+@pytest.mark.parametrize("model_class", [TQC, QRDQN])
 def test_cnn(tmp_path, model_class):
     SAVE_NAME = "cnn_model.zip"
     # Fake grayscale with frameskip
     # Atari after preprocessing: 84x84x1, here we are using lower resolution
     # to check that the network handle it automatically
     env = FakeImageEnv(screen_height=40, screen_width=40, n_channels=1, discrete=model_class not in {TQC})
     kwargs = {}
-    if model_class in {TQC}:
+    if model_class in {TQC, QRDQN}:
```

Comment:
we should probably also reduce the number of quantiles of QR-QDN during the tests, so they run faster, no?
(QR-DQN uses 200 quantiles by default vs 25 for TQC)